source/tests/consistent/descriptor/test_dpa2.py

```
@@ -201,8 +271,8 @@ def skip_pt(self) -> bool:
             repinit_set_davg_zero,
             repinit_type_one_side,
             repinit_use_three_body,
-            repformer_update_g1_has_conv,
             repformer_direct_dist,
+            repformer_update_g1_has_conv,
```

Ruff (0.15.9)

[warning] Unpacked variable `repformer_update_g1_has_conv` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)